dom/media/VideoFrameContainer.cpp

```diff
@@ -153,6 +154,8 @@ void VideoFrameContainer::SetCurrentFramesLocked(
   //  until it is safe.
   nsTArray<ImageContainer::OwningImage> oldImages;
   mImageContainer->GetCurrentImages(&oldImages);
+  recordreplay::RecordReplayAssert("[RUN-1253] VideoFrameContainer::SetCurrentFramesLocked #2 %u",
```

Comment:
Why do you use `%u` + casting instead of `%d`?

Reply:
The casting is just to make sure that we're seeing int-sized things instead of long-sized things on platforms where it matters (I'm paranoid).  The '%u' is coming from the fact that the underlying `.Length()` is going to be unsigned anyway, but it could go either way.